### DISCUSSION TITLE
osu!dev catchup meetings being recorded and posted on youtube.

### DISCUSSION THREAD

**Discussion by estrogenn** [ppy/osu#27824] [Ideas] [answered]
would this be something the osu!team would do, or does anyone else want this besides me?  I would be willing to do this by myself if I needed to.  Maybe a discord channel would also be made with the videos linked etc. If not its totally fine I just know a lot of people miss it due to time zone differences, thanks.

**Comment by peppy** [comment, marked as answer]
not going to happen. those are nto suposed to be recorded meetings and there will be no benefit from doing so.

**Reply by Rekunan** [reply]
Adding on, I've found that usually you can predict what's going to be discussed in the meetings if you follow the respective ppy repositories, and anything extra/future will eventually pop up in one if not already (unless if scrapped or whatever).
